src/main/java/appeng/integration/modules/RC.java

```diff
@@ -49,13 +52,10 @@ public void rockCrusher( ItemStack input, ItemStack output )
 	@Override
 	public void init()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)